x/upgrade/keeper/migrations.go

```diff
@@ -42,3 +48,27 @@ func migrateDoneUpgradeKeys(ctx sdk.Context, storeKey storetypes.StoreKey) error
 	}
 	return nil
 }
+
+func (m Migrator) Migrate2to3(ctx sdk.Context) error {
```

Comment:
Can we add a simple godoc for this stating what it does pls?

Reply:
added